include/internal/catch_test_case_tracker.cpp

```diff
@@ -190,7 +191,7 @@ namespace TestCaseTracking {
 
         if ((m_filters.empty() || m_filters[0] == "") ||
              std::find(m_filters.begin(), m_filters.end(),
-                       m_nameAndLocation.name) != m_filters.end())
+                       trim(m_nameAndLocation.name)) != m_filters.end())
```

Comment:
Similarly, why isn't this trimmed at construction, rather than on each call to `isComplete`?